python/ray/experimental/workflow/examples/comparisons/metaflow/foreach_metaflow.py.txt

```diff
@@ -0,0 +1,27 @@
+# https://docs.metaflow.org/metaflow/basics#foreach
+from metaflow import FlowSpec, step
```

Comment:
change the file name to `.py` ?

Reply:
I didn't want this to get picked up by the automated testing (otherwise we'd need to install metaflow).